src/mono/sample/wasm/browser-bench/Wasm.Browser.Bench.Sample.csproj

```diff
@@ -17,29 +17,22 @@
     <WasmExtraFilesToDeploy Include="frame-main.js" />
     <WasmExtraFilesToDeploy Include="style.css" />
     <Compile Remove="Console/Console.cs" />
+    <PackageReference Include="Microsoft.Build.Utilities.Core" Version="$(MicrosoftBuildUtilitiesCoreVersion)" ExcludeAssets="runtime" />
   </ItemGroup>
 
   <Target Name="RunSample" DependsOnTargets="RunSampleWithBrowserAndSimpleServer" />
+  <UsingTask TaskName="GetNugetConfigTask" AssemblyFile="$(OutputPath)Wasm.Browser.Bench.Sample.dll" />
+
+  <Target Name="SetNugetConfigContent" BeforeTargets="PrepareBlazorTemplate;BuildBlazorFrame">
```

Comment:
`BeforeTargets` should contain `PrepareBrowserTemplate`, `BuildBlazorFrame` doesn't need to be here.

Reply:
It's `BuildBlazorFrame` -> `PrepareBlazorTemplate` -> `NugetConfigContent`.